gatsby-node.js

```diff
@@ -66,4 +66,9 @@ exports.createPages = ({ actions }) => {
     toPath: '/developing/vue-tutorial/overview',
     isPermanent: true,
   });
+    createRedirect({ 
+     fromPath: '/guidelines/typography/styling-strategies', 
+     toPath: '/guidelines/typography/style-strategies', 
+     isPermanent: true, 
+   });
```

Comment:
Sorry just some minor formatting here that I can suggest and commit

```suggestion
  createRedirect({
    fromPath: '/guidelines/typography/styling-strategies',
    toPath: '/guidelines/typography/style-strategies', 
    isPermanent: true,
  });
```